package/main/daml/Daml.Finance.Interface.Types.Date/daml.yaml

```diff
@@ -4,7 +4,7 @@
 sdk-version: 2.7.0
 name: daml-finance-interface-types-date
 source: daml
-version: 2.0.1
+version: 2.0.2
```

Comment:
Are these version bumps required for the project to build? It shouldn't be the case given that we bumped almost all packages with the latest release.

Reply:
Seems so, I tried to unbump it, but it fails both locally and in the CI: https://github.com/digital-asset/daml-finance/pull/1024/commits/035c1456e517104ddef1a5f41cc951132199579b

Reply:
👍 thanks for checking

Reply:
Seems it was 2.0.1 that we released in v2.7, so given these current changes I think it makes sense that a bump is required now: https://docs.daml.com/daml-finance/#:~:text=Interface.Types.Date-,2.0.1,-Stable